README.md

```diff
@@ -1,64 +1,65 @@
 [![Contribution Guidelines](http://img.shields.io/badge/CONTRIBUTING-Guidelines-blue.svg)](./.github/CONTRIBUTING.md)
 
 Jump in: [![Slack Status](http://slack.samvera.org/badge.svg)](http://slack.samvera.org/)
 
-<!-- TOC depthFrom:1 depthTo:6 withLinks:1 updateOnSave:1 orderedList:0 -->
-# Table of Contents
-- [Introduction](#introduction)
-  - [Documentation](#documentation)
-  - [Wiki](https://github.com/marriott-library/newspaper_works/wiki)
-  - [Features Matrix](https://github.com/marriott-library/newspaper_works/wiki/Features-Matrix)
+<!-- TOC -->
+
 - [Overview](#overview)
-	- [Purpose, Use, and Aims](#purpose-use-and-aims)
-	- [Development Status](#development-status)
-	- [Requirements](#requirements)
-	  - [Newspaper Works Dependencies](#newspaper-works-dependencies)
-	- [PCDM metadata Model](#Newspapers-PCDM-metadata-model)
-- [Installation/Testing](#installationtesting)
-	- [Extending, Using](#extending-using)
-	- [Basic Model Use (console)](#basic-model-use-console)
-	- [Application/Site Specific Configuration](#applicationsite-specific-configuration)
-	- [Development and Testing Setup](#development-and-testing-setup)
-- [Credits](#credits)
-	- [Sponsoring Organizations](#sponsoring-organizations)
-	- [Contributors and Project Team](#contributors-and-project-team)
-	- [More Information / Contact](#more-information-contact)
-<!-- /TOC -->
-# Introduction
-The Newspapers in Samvera is an IMLS grant funded project to develop newspaper specific functionality for the [Samvera](http://samvera.org/) Hyrax framework.
+  - [Documentation](#documentation)
+  - [Requirements](#requirements)
+  - [Dependencies](#dependencies)
+- [Installation](#installation)
+  - [Application/Site Specific Configuration](#applicationsite-specific-configuration)
+    - [Config changes made by the installer:](#config-changes-made-by-the-installer)
+    - [Configuration changes you should make after running the installer](#configuration-changes-you-should-make-after-running-the-installer)
+- [Ingesting Content](#ingesting-content)
+- [Developing, Testing, and Contributing](#developing-testing-and-contributing)
+  - [Contributing](#contributing) 
+- [Acknowledgements](#acknowledgements)
+  - [Sponsoring Organizations](#sponsoring-organizations)
+  - [More Information](#more-information)
+  - [Contact](#contact)
 
-## Documentation
-We are currently working on adding and updating documentation on our [Project Wiki](https://github.com/marriott-library/newspaper_works/wiki)
+<!-- /TOC -->
 
 # Overview
-The Newspaper Works gem provides work type models and administrative functionality for Hyrax-based Samvera applications in the space of scanned newspaper media.  This gem can be included in a Digital Asset Management application based on Hyrax 2.5.1
+NewspaperWorks is a gem (Rails "engine") for [Hyrax](http://hyrax.samvera.org/) -based digital repository applications to support ingest, management, and display of digitzed newspaper content.
```

Comment:
Update hyrax link to https

Reply:
Fixed in https://github.com/marriott-library/newspaper_works/pull/184/commits/273880d4612d69bd46c9817b6ca4523fa669826e